src/data-tutorials/users.tsx

```diff
@@ -845,6 +845,13 @@ const Users: User[] = [
     website: 'https://medium.com/celodevelopers/introduction-to-creating-nfts-on-celo-eb7240a71cc0',
     tags: ['beginner','foundation', 'nft', 'erc721'],
```

Comment:
Use the advised tags.

Reply:
Can you kindly elaborate on the advised tags and the correlation with my tutorial, just for clarity.

Reply:
Ignore the 'erc721' tag

Reply:
what about the nft tag?

Reply:
sure, you can too. Make sure to add the celosage tag in your article's frontmatter and src/data-tutorials/users.tsx file.

Kindly checkout out this for [best practices](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices)

Reply:
And finally there is something wrong with your deployment, kindly check it out